app/Http/Controllers/CompetitionsController.php

```diff
@@ -39,7 +48,9 @@ public function show(Competition $competition)
     {
         $contest = Contest::find($competition->contest_id);
 
-        return view('competitions.show', compact('competition', 'contest'));
+        $users = $this->repository->getAll($competition->users->pluck('id')->toArray());
```

Comment:
We actually do want an array here, so all is well @angaither! Alternatively could use the `->all()` method on the collection which just grabs the items property.